modules/thermal_hydraulics/src/fluidproperties/LinearFluidProperties.C

```diff
@@ -84,7 +88,11 @@ LinearFluidProperties::c_from_v_e(Real v, Real e, Real & c, Real & dc_dv, Real &
   dc_de = 0;
 }
 
-Real LinearFluidProperties::cp_from_v_e(Real, Real) const { return 0; }
+Real
+LinearFluidProperties::cp_from_v_e(Real, Real) const
+{
+  return _cv;
```

Comment:
Pr, mu, and k are user parameters (though they're optional...), so this should probably be computed as `k*Pr/mu`.

Reply:
I see you deprecated `Pr`, so this is resolved 👍